apps/backend-e2e/src/map-review.e2e-spec.ts

```diff
@@ -278,14 +278,15 @@ describe('Map Reviews', () => {
       it('should 401 when no access token is provided', () =>
         req.unauthorizedTest('map-review/1', 'patch'));
     });
+
     describe('DELETE', () => {
       let user, token, u2Token, modToken, map, review;
       const assetPath = mapReviewAssetPath('1');
 
       beforeAll(async () => {
-        [[user, token], modToken] = await Promise.all([
-          db.createAndLoginUser(),
+        [[user, token], u2Token, modToken] = await Promise.all([
```

Comment:
This changes should probably be in https://github.com/momentum-mod/website/pull/905/commits/c9d041c2348fd9aea34f158160fa5eb5dc1bd590

Reply:
Fixed. Though IMO this isn't worth bringing up, since it's purely a Git thing and a lot of people would know how to fix this easily. Would be curious what Nick and any others think

Reply:
I agree, but when your split change to commits based on what is done in them it gets a bit confusing where did these lines come from.